backend/SeedData/SampleUsers.cs

```diff
@@ -0,0 +1,54 @@
+using Microsoft.AspNetCore.Identity;
+using WhaleSpotting.Models.Data;
+
+namespace WhaleSpotting.Data;
+
+public static class SampleUsers
+{
+    public const int NumberOfUsers = 2;
+
+    private static readonly IList<IList<string>> DataUsers = new List<IList<string>>
+    {
+        new List<string> { "Sam", "White", "sam1", "user1@gmail.com", "Pa$$word1" },
+        new List<string> { "Evie", "Brown", "evie2", "user2@gmail.com", "Pa$$word2" },
+    };
+
+    public static async Task CreateAdminAsync(UserManager<User> userManager)
+    {
+        var poweruser = new User
```

Comment:
I think `poweruser` should be changed to `adminUser` to align with the role type and some of the other method names.
please can you change that and the other references to power to be admin?